### PR TITLE
Issue/888

### PR DESCRIPTION
**Pull request by micahnyc**

### CODE REVIEW

**Comment by micahnyc** [comment]
This adds layers from treks to openspace

741 for Moon
157 for Mars
23 for Mercury

File: data/assets/scene/solarsystem/planets/earth/moon/nasa-treks/Apollo_15_Metric_Cam_DEM_ColorHillshade/Apollo_15_Metric_Cam_DEM_ColorHillshade.asset [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  asset.onDeinitialize(function()
+    openspace.globebrowsing.deleteLayer(globeIdentifier, "ColorLayers", layer.Identifier)
+  end)
+  
+  asset.export("layer", layer)
+
+
+  asset.meta = {
+    Name = [[Apollo 15 Metric Cam DEM, ColorHillshade]],
+    Version = '1.0',
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/planets/earth/moon/nasa-treks/Apollo_15_Metric_Cam_DEM_ColorHillshade/Apollo_15_Metric_Cam_DEM_ColorHillshade.asset:23]: For consistency, these should be `"` instead

File: data/assets/scene/solarsystem/planets/earth/moon/nasa-treks/Apollo_15_Metric_Cam_DEM_ColorHillshade/Apollo_15_Metric_Cam_DEM_ColorHillshade.asset [hunk 2]
```diff
@@ -0,0 +1,28 @@
+local globeIdentifier = asset.require("scene/solarsystem//planets/earth/moon/moon").Moon.Identifier
+
+  local layer = {
+    Identifier = "Apollo15_MetricCam_ClrShade_Global_1024ppd",
+    Name = [[Apollo 15 Metric Cam DEM, ColorHillshade]],
+    FilePath = asset.localResource("Apollo15_MetricCam_ClrShade_Global_1024ppd.vrt"),
+    Description = [[undefined]]
```
Comment (alexanderbock) [inline data/assets/scene/solarsystem/planets/earth/moon/nasa-treks/Apollo_15_Metric_Cam_DEM_ColorHillshade/Apollo_15_Metric_Cam_DEM_ColorHillshade.asset:7]: "undefined"

**Review by alexanderbock — APPROVED** [review]
Tiny adjustments for the script but otherwise good to go :+1:

**Pull request merged**